examples/java/spring-boot/build.gradle

```
@@ -0,0 +1,8 @@
+plugins {
+  id 'uk.gov.api.spring-boot-conventions'
+}
+
+dependencies {
+  testImplementation 'org.springframework.boot:spring-boot-starter-test'
+  implementation 'org.springframework.boot:spring-boot-starter-web'
```

Can we swap these around please? It reads a bit easier to see `api` then `implementation`, then anything `test*`
```suggestion
  implementation 'org.springframework.boot:spring-boot-starter-web'
  testImplementation 'org.springframework.boot:spring-boot-starter-test'
```